client/src/Components/local/atoms/core-mapping/helper/core-renderer-helper.tsx

```diff
@@ -0,0 +1,18 @@
+import { FeatureCollection, GeoJsonProperties, Geometry } from 'geojson'
+import { RENDERER_CORE, RENDERER_MILSYM } from 'src/custom-types'
+import { CoreRenderer } from '../renderers/core-renderer'
+import { MilSymbolRenderer } from '../renderers/milsymbol-renderer'
+import { ICoreRenderer } from '../renderers/I-core-renderer'
+
+export class CoreRendererHelper {  
+  static from (type: string, features: FeatureCollection<Geometry, GeoJsonProperties>): ICoreRenderer {
+    switch (type) {
+      case RENDERER_CORE:
```

Comment:
Hello @lebaphi - in the next line, why don't we just pass `features` with the constructor.  We wouldn't need `from` then.  Or is this a programming practice that's new to me?